### DISCUSSION TITLE
Missing Chia Holiday 2021 CH21 Token

### DISCUSSION THREAD

**Discussion by ErdalKELES** [Chia-Network/chia-blockchain#11350] [Support]
Hello,

I have been doing chia token farm for about 1 year. I installed the 1.3.4 version on my computer that I farmed and did not do. No Chia Holiday 2021 (CH21) tokens awarded as rewards. My farm has found 6-7 Chia tokens.

05/2021-06-2021 Hpools 2 XCH Old Plots
06/2021-07-2021 Solo 1 XCH
07-2021-04-2022 Space pool 4 XCH

How to get gift tokens please help.

**Comment by loppefaaret** [comment]
If you want to, you can tell me what address you have been farming blocks wins with, and i can double check if we have it in our internal document of rewarded farmer addresses - if you indeed have not received your CH21 tokens, we will make sure that you will.
If you tel lme what "farmer rewards target address" you have been using, i will check

**Reply by qaqz5255** [reply]
I updated version 1.3.4 to have a Chinese holiday 2021 (ch21) token reward. I found two Kia tokens on my farm. It is found that the processing is successful, but it is not displayed on other platforms. What's the matter? My is Windows platform.

**Reply by ErdalKELES** [reply]
Hello,
My Gate.io wallet : xch19ndd9lwgwlnx538jl9gl2x4tw7uurldeze938ls36k20h8fppcmspmymc3
Huobi wallet : xch157r52hf42zld4fdxg6652d5zq2x5d94pa2tz23ax00mqua5e0wfqpdccnq
1 mojo and first chia software wallet address : xch1qy63dzvwm420zs3v58cvgms3kc9htfnl9uxhvmnue7nc8jjy3pfseg33ex
i am use 2 wallet address.

**Reply by qaqz5255** [reply]
I use OKx wallet

**Reply by qaqz5255** [reply]
I extracted two 21ch's into the OKx wallet. Chia's prompt has been handled, but I didn't receive it in the OKx wallet. Do you know what's going on?

**Reply by loppefaaret** [reply]
> I extracted two 21ch's into the OKx wallet. Chia's prompt has been handled, but I didn't receive it in the OKx wallet. Do you know what's going on?

I'm pretty sure that if you ask okx to deposit XCH, and you send them CH21 tokens - they are lost, unless you can convince okx support to return the tokens

**Reply by loppefaaret** [reply]
> Hello, My Gate.io wallet : xch19ndd9lwgwlnx538jl9gl2x4tw7uurldeze938ls36k20h8fppcmspmymc3 Huobi wallet : xch157r52hf42zld4fdxg6652d5zq2x5d94pa2tz23ax00mqua5e0wfqpdccnq 1 mojo and first chia software wallet address : xch1qy63dzvwm420zs3v58cvgms3kc9htfnl9uxhvmnue7nc8jjy3pfseg33ex i am use 2 wallet address.

the houbi address is listed as a CH21 token holder, it was rewarded 2 CH21 tokens `xch157r52hf42zld4fdxg6652d5zq2x5d94pa2tz23ax00mqua5e0wfqpdccnq`

**Reply by ErdalKELES** [reply]
Hello, How can I get this award? This coin does not appear in Huobi wallets.

**Reply by loppefaaret** [reply]
You should not have chosen to direct your farming rewards directly into a wallet that you do not control - unless huobi wants to help you get your tokens, they are lost

**Reply by ErdalKELES** [reply]
> You should not have chosen to direct your farming rewards directly into a wallet that you do not control - unless huobi wants to help you get your tokens, they are lost

Please TX ID :((

**Reply by ErdalKELES** [reply]
> > Hello, My Gate.io wallet : xch19ndd9lwgwlnx538jl9gl2x4tw7uurldeze938ls36k20h8fppcmspmymc3 Huobi wallet : xch157r52hf42zld4fdxg6652d5zq2x5d94pa2tz23ax00mqua5e0wfqpdccnq 1 mojo and first chia software wallet address : xch1qy63dzvwm420zs3v58cvgms3kc9htfnl9uxhvmnue7nc8jjy3pfseg33ex i am use 2 wallet address.
> 
> the houbi address is listed as a CH21 token holder, it was rewarded 2 CH21 tokens `xch157r52hf42zld4fdxg6652d5zq2x5d94pa2tz23ax00mqua5e0wfqpdccnq`

Huobi support recomented to TX ID. Please 2 CH21 TX ID.

**Reply by ErdalKELES** [reply]
Hello @loppefaaret,
Any chance to send a TXID for the 2 ch21 tokens I won?

@loppefaaret say;
the houbi address is listed as a CH21 token holder, it was rewarded 2 CH21 tokens xch157r52hf42zld4fdxg6652d5zq2x5d94pa2tz23ax00mqua5e0wfqpdccnq

**Comment by qaqz5255** [comment]
I updated version 1.3.4 to have 2 Chia holiday 2021 ch21 tokens, but after withdrawing, I found that the processing was successful, but it was not displayed on other platforms. What's the matter? My is Windows platform.

**Reply by loppefaaret** [reply]
is the other platform that is not showing, okx ? i doubt they support tokens ? and only supports XCH

**Comment by tonyx888** [comment]
I have been farming chia for 1 year, half with hpool OG and other with space pool official protocol, I have not received any tokens, what should I do?

**Reply by loppefaaret** [reply]
tell me what your "farmer rewards target address" is, and ill check wether it qualifies, and if it should have received CH21 tokens

**Reply by ErdalKELES** [reply]
i am check to internet address? only you?

**Reply by loppefaaret** [reply]
I'm sorry, but your only address that was eligible for the CH21 tokens, where the address you said came from houbi - so it does not belong to you, but is controlled by houbi

**Reply by ErdalKELES** [reply]
i am check another wallet. Only you can do this query.
